src/main/java/uk/gov/ons/census/casesvc/service/FieldworkFollowupService.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.ons.census.casesvc.service;
+
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import uk.gov.ons.census.casesvc.model.dto.FieldWorkFollowup;
+import uk.gov.ons.census.casesvc.model.entity.Case;
+
+@Component
+public class FieldworkFollowupService {
```

Comment:
This whole thing should not be in Case Processor for 2021 at all. We need to simplify Case Processor. CCS was hacked in at the end of rehearsal. It's time to hack it out and do it again properly.

Reply:
There's an almost identical class to this that's been left in Case Processor called `CcsToFieldService`